Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -237,17 +179,7 @@ private static async Task<string> FindMsBuildPathAsync()
             return string.Empty;
         }
 
-        public static async Task<bool> RestoreNugetPackagesAsync(string nugetPath, string storePath)
-        {
-            Debug.Assert(File.Exists(nugetPath));
-            Debug.Assert(Directory.Exists(storePath));
-
-            await new Process().StartProcessAsync(nugetPath, $"restore \"{storePath}/project.json\"");
-
-            return File.Exists($"{storePath}\\project.lock.json"); ;
-        }
-
-        private static bool SetPackageVersion(bool increment)
+        private static bool UpdateAppxManifest(IBuildInfo buildInfo)
         {
             // Find the manifest, assume the one we want is the first one
             string[] manifests = Directory.GetFiles(BuildDeployPreferences.AbsoluteBuildDirectory, "Package.appxmanifest", SearchOption.AllDirectories);
```

Comment:
I've tried to run the changes on this branch on Azure Pipelines. When I do so, the value of `BuildDeployPreferences.AbsoluteBuildDirectory`  is a nonexistent directory found in the root called _Builds_. Any idea why this may be happening?

In [my PR](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3615) I amended this by passing `buildDirectory` to this method from `BuildAppxAsync`. Can you do something similar here (pass `buildInfo.OutputDirectory`)?

Reply:
That's odd, it should be creating that directory automatically if it doesn't exist. I'll double check this.

Reply:
Is there a way that we can specify this directory? Should it not be `-buildOutput `

Reply:
Sure, we could do that. We'll add the path to the argument switch above where we're building the buildInfo object